### PR TITLE
Ahammed Imthias

### PR DESCRIPTION
**Pull request by lalu-windows**
Added my card to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <div class="card">
```
Comment (TenTraicion) [inline index.html:118]:
```suggestion
        <!-- ________*Ahammed* Contributor card START ________ -->
        <div class="card">
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
+              </li>
+            </ul>
+          </div>
+        </div>
```
Comment (TenTraicion) [inline index.html:139]:
```suggestion
        </div>
        <!-- ________*Ahammed* Contributor card END ________ -->
```

File: index.html [hunk 3]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <div class="card">
+          <p class="name">Ahammed Imthias</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="https://twitter.com/LeviAckerman__l">Levi_ackerman_l</a>
```
Comment (TenTraicion) [inline index.html:122]:
```suggestion
            <a href="https://www.twitter.com/LeviAckerman__l" target="_blank">@Levi_ackerman_l</a>
```

File: index.html [hunk 4]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">about you</p>
```
Comment (TenTraicion) [inline index.html:124]: Write something about your self here...
```suggestion
          <p class="about">about you</p>
```

File: index.html [hunk 5]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
...
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```
Comment (TenTraicion) [inline index.html:129]: add 3 resources that can help felow developers with a link in place of `#`, a suitable title by replacing `First Resource` and the resource name in the place where `Resource 1` is written.
```suggestion
                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```

File: index.html [hunk 6]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
```
Comment (TenTraicion) [inline index.html:132]: add the second resource here

File: index.html [hunk 7]
```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
```
Comment (TenTraicion) [inline index.html:135]: complete your card by adding the third resource here.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@lalu-windows, kindly do the recommended changes and commit. 

for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

**Comment by TenTraicion** [comment]
@lalu-windows, kindly do the recommended changes. The PR will be closed if we get no response within 7 days of opening. Thanks,

**Comment by CandidDeer** [comment]
@lalu-windows Closing this PR now due to non-response. Please know you can create a new PR anytime and we'll be happy to merge it as long as it follows the guidelines. Thx.

**Pull request closed without merging**